src/mongocxx/test/spec/util.cpp

```diff
@@ -222,11 +222,16 @@ void set_up_collection(const client& client,
     auto db = client[test[database_name].get_string().value];
     db.drop();
 
+    bsoncxx::builder::basic::document opts;
+    if (test["encrypted_fields"]) {
+        opts.append(kvp("encryptedFields", test["encrypted_fields"].get_document().value));
+    }
```

Comment:
```suggestion
    bsoncxx::builder::basic::document opts;
    if (const auto fields = test["encrypted_fields"]) {
        opts.append(kvp("encryptedFields", fields.get_document().value));
    }
```
Avoid double-lookup.